lib/ios_analytics/version.rb

```diff
@@ -1,3 +1,3 @@
 module IosAnalytics
-  VERSION = "0.0.4"
+  VERSION = "0.0.5"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)